lib/mayaUsd/render/vp2RenderDelegate/basisCurves.cpp

```diff
@@ -1280,6 +1287,7 @@ HdVP2BasisCurves::_UpdateDrawItem(
         // If available, something changed
         if (stateToCommit._shader != nullptr) {
             renderItem->setShader(stateToCommit._shader);
+            renderItem->setWantDiffuseColorMultiDrawConsolidation(stateToCommit._isFallbackShader);
```

Comment:
I guess it is probably ok for now to not enable color consolidation for curves with material assignment, but at some point we will need to do it.